### ISSUE TITLE
Steel Diver: Sub Wars

### ISSUE THREAD

**Issue by notprogramminggames** [Panda3DS-emu/Panda3DS-Games-List#91]
### Game Name (and version if necessary)

Steel Diver: Sub Wars 

### Operating System

Windows 11 22H2

### Game State

Water textures flicker a lot and the screen goes pink under water. Sound seems to not work

### Screenshots

![kcHzVoYYMJ](https://github.com/Panda3DS-emu/Panda3DS-Games-List/assets/148684723/825feaab-8b3d-4709-9615-5c59b914db3b)
![QLYwNzYC0W](https://github.com/Panda3DS-emu/Panda3DS-Games-List/assets/148684723/3ab2238c-1206-49ab-9eef-1d135e5a6064)
![ePa07IGYJg](https://github.com/Panda3DS-emu/Panda3DS-Games-List/assets/148684723/18a152fe-342f-46b0-831d-fc8aae1d2d53)

**Comment by wheremyfoodat** [comment]
Can you try a nightly build? Mii Selector should work there.

**Comment by notprogramminggames** [comment]
Yep it works now. Do you know if there is a way to show the bottom screen without part of it being unseeable?

**Comment by wheremyfoodat** [comment]
This might be a fuckup in the GUI code in the Qt frontend, I'll need to check. The messed up textures issue should be a shader bug, it sounds vaguely familiar.

The missing sound is cause sound is not implemented yet in general.
